src/Stores/Politicians/reducer.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+import type { PoliticianType, Action } from '../../types';
+
+export type State = {
+  +politicians: ?Array<PoliticianType>,
+  +loading: boolean,
+  +error: *,
+};
+
+const initialState = {
+  politicians: null,
```

Comment:
I'm all for using null as the initial state.

That would then convey the state "Has not tried to fetch politicians yet"
An empty array would convey the state "I fetched politicians, but didn't find any"